src/django/api/models.py

```diff
@@ -85,6 +86,33 @@ def create_superuser(self, email, password, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
 
+class ContributorManager(models.Manager):
+    TRIGRAM_SIMILARY_THRESHOLD = 0.5
```

Comment:
This was chosen via trial an error with development data. We may adjust this in the future if non enough matching occurs or an excessive number of mismatches need to be corrected.